src/opengl_render.c

```diff
@@ -665,6 +665,38 @@ void gl_drawCircle( const double cx, const double cy,
 }
 
 
+/**
+ * @brief Draws a line.
+ *
+ *    @param cx x1 position of the first point in screen coordinates.
+ *    @param cy y1 position of the first point in screen coordinates.
+ *    @param cx x1 position of the second point in screen coordinates.
+ *    @param cy y1 position of the second point in screen coordinates.
+ *    @param c Colour to use.
+ */
+void gl_drawLine( const double x1, const double y1,
+      const double x2, const double y2, const glColour *c )
+{
+   GLfloat vertex[4];
+   gl_Matrix4 projection;
+   gl_vbo *my_vbo;
+
+   projection = gl_view_matrix;
+   vertex[0] = x1;
+   vertex[1] = y1;
+   vertex[2] = x2;
+   vertex[3] = y2;
+
+   my_vbo = gl_vboCreateStatic( sizeof(GLfloat) * 4, vertex );
```

Comment:
It's probably better to create a global VBO and modify it here; I think creating and destroying VBOs regularly isn't very efficient.